### ISSUE TITLE
Post install message

### ISSUE THREAD

**Issue by cyberious** [voxpupuli/metadata-json-lint#90]
Remove annoying post install message, instead we should move to a message when the puppet version is actually meets the criteria. It tends to be very confusing for users if they are getting this message despite being on a greater version that listed.

https://github.com/voxpupuli/metadata-json-lint/blob/master/metadata-json-lint.gemspec#L25-L31

**Comment by rnelson0** [comment]
I agree, but I am not experienced enough to know how to detect that in a setup, whether using system gems or bundler, to display the message appropriately.

**Comment by bwitt** [comment]
the post install message is gone, so this can be closed.
